### DISCUSSION TITLE
SHIFT+CLIC Button

### DISCUSSION THREAD

**Discussion by ghost** [noah-nuebling/mac-mouse-fix#313] [Questions] [answered]
Hello!

I'm required to select a looooooot mean I need to hold a button on the keyboard or 3D Space mouse and click, click and click! 
I have experienced Logitech option softawre but it became too annoying. The associated changes get lost randomly and ask me to restart my computer, yes Logitech option ask me randomly to close all my apps and restart my computer.

So Im ready, what mouse should I get to have my  SHIFT+Click button, please save my left hand!

**Comment by ghost** [comment]
MX Master 3 is my actual Mouse. Button 6 and 7 can't be used with MMF.

**Comment by ghost** [comment]
Asus Pro ART MD300?

**Comment by MuhammedZakir** [comment]
Why not use a remapper? Karabiner-Elements, Hammerspoon, BetterTouchTool(?), etc. should enable you to do this with *any* mouse.

**Comment by noah-nuebling** [comment, marked as answer]
Hi!
Marking this as answered since I'm moving the topic to Issue #358. Please continue discussion there. Thanks!
